ucfs-claimant-api-mysql-interface.tf

```diff
@@ -54,3 +54,21 @@ resource "null_resource" "ucfs_claimant_api_mysql_interface" {
     command = "./initial-commit.sh ${github_repository.ucfs_claimant_api_mysql_interface.name} '${github_repository.ucfs_claimant_api_mysql_interface.description}' ${github_repository.ucfs_claimant_api_mysql_interface.template.0.repository}"
   }
 }
+
+resource "github_actions_secret" "ucfs-claimant-api-mysql-interface_github_email" {
+  repository      = github_repository.ucfs_claimant_api_mysql_interface.name
+  secret_name     = "GITHUB_EMAIL"
+  plaintext_value = var.github_email
+}
+
+resource "github_actions_secret" "ucfs-claimant-api-mysql-interface_github_username" {
+  repository      = github_repository.ucfs_claimant_api_mysql_interface.name
+  secret_name     = "GITHUB_USERNAME"
+  plaintext_value = var.github_username
+}
+
+resource "github_actions_secret" "ucfs-claimant-api-mysql-interface_github_token" {
+  repository      = github_repository.ucfs_claimant_api_mysql_interface.name
+  secret_name     = "GITHUB_TOKEN"
+  plaintext_value = var.github_token
+}
```

Comment:
this isn't needed/used

Reply:
May be required in the future. I'd argue we should supply these secrets to all repos if they can't be accessed externally.